util/fiber_socket_base.h

```diff
@@ -196,6 +196,10 @@ class LinuxSocketBase : public FiberSocketBase {
   int32_t fd_;
 };
 
+void SetNonBlocking(int fd);
```

Comment:
we call it only from epoll_socket (which is both for linux/epoll and macos/freebsd). Lets declare and implement those methods inside epoll_socket.cc in anonymous namespace.

Reply:
Its also used by `LinuxSocketBase::Create` - could update `EpollSocket` to override `Create`?

Reply:
no, it's fine.